src/components/policy/policy_regular/src/cache_manager.cc

```diff
@@ -585,6 +585,13 @@ bool CacheManager::SecondsBetweenRetries(std::vector<int>& seconds) {
 
   const std::size_t size =
       pt_->policy_table.module_config.seconds_between_retries.size();
+
+  const std::size_t max_seconds_between_retries_size = 5u;
+  if (size > max_seconds_between_retries_size) {
+    // must exit app
+    exit(EXIT_FAILURE);
```

Comment:
I suppose that you should validate length of this array in policy table validation and return is_valid  = falce. And in LifeCycle start you shuld check if policy table validation failed SDL should not be started with appropriate error message